.gitmodules

```diff
@@ -3,4 +3,12 @@
 	url = https://github.com/foundry-rs/forge-std
 [submodule "lib/forge-std"]
 	branch = v1.2.0
```

Comment:
do you know why this is in here twice?

Reply:
removed the duplicated e93a06fd171c37a421ae4fd62c965adf5dde7ee3

Reply:
got it, assuming for version control for now it is fine to take the latest head of main?